### PR TITLE
Trn 82 demo of the language translations

### PR DESCRIPTION
**Pull request by rosamakinen**
_[ the texts for the translations are still temp, so just to demo the effect, and will be changed later]_

**Framework for language translations in index.js file**
- Custom event listener for when view gets updated, we use it to figure out when we need a new view translated
- Storing required page and language in localStorage
- Reading translations from JSON files and appending translations to the elements textContent

**-Translation files in /static/translations/, named same as the page to be translated**

**-Buttons to switch between translations are now in nav bar** _(maybe this should be made into a dropdown and the option to switch could also be added to settings as well)_

**-Set up files to translate the existing pages** (except for login/register/friends & pong, since I did not want to work on same files as Sunny/Rasmus to not get merge issues)

**-Added settings -page** to have the frame for translations ready and added tiny bit of information on it, this needs to be expanded to it's full capacity. 

**In order to do translations for a page:** 
-have a JSON file named the same as the page/ view script.
-in getHTML add a "page" and name of the page to the local storage, so we can find the translation.
-for each element to be translated addAttribute "lang-key", for the key value, add a descriptive word to use as a variable for the json file.
-for the json files use the key value as a key in the json file, and have it's value be the translated text. 
(examples for each step can be found in the code)

Moving forward, for each page we would from the get go build the translations as well.

### CODE REVIEW

File: frontend/static/js/index.js
Comment (RushMaverick) [inline frontend/static/js/index.js:1]: So is localStorage somewhere where files are saved for further use?

File: frontend/static/js/views/AView.js [hunk 1]
```diff
@@ -31,42 +31,66 @@ export default class {
...
-        link.href = href;
-        link.textContent = text;
-        link.dataset.link = true;
-        p.appendChild(link);
-        if (additionalText) {
-            const span = document.createElement('span');
-            span.textContent = additionalText;
-            p.appendChild(span);
-        }
+		p.textContent = 'ei vieläkään';
```
Comment (RushMaverick) [inline frontend/static/js/views/AView.js:53]: If this is just test stuff, make sure to remove it when you are done.

File: frontend/static/js/index.js
Comment (rosamakinen) [reply frontend/static/js/index.js:1]: No, not the Files them selves, but information on which files to read.

**Review by RushMaverick — APPROVED** [review]
Awesome, great job Rosa! I saw you changed to single quotes as well so I will strive to change that in my code to fit. 
If you could, can I ask you to write the instructions down in the appropriate ticket for Confluence as well? 
Thanks!

**Comment by rosamakinen** [comment]
Yes! I will make confulence documentation for how to make translations  as well! Planning to do that remote on friday 💪🏻

**Review by jestebanpelaez18 — APPROVED** [review]
Good job! I really want to see this feature in our project!

**Pull request merged**